tools/src/test/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommandTestUtils.java

```diff
@@ -68,15 +68,31 @@ static List<ClusterConfig> generator() {
         ClusterConfig consumerGroupCoordinator = ClusterConfig.defaultBuilder()
                 .setTypes(Stream.of(KRAFT, CO_KRAFT).collect(Collectors.toSet()))
                 .setServerProperties(serverProperties)
-                .setTags(Collections.singletonList("newGroupCoordinator"))
+                .setTags(Collections.singletonList("consumerGroupCoordinator"))
                 .build();
         return Arrays.asList(classicGroupCoordinator, consumerGroupCoordinator);
     }
 
+    static <T> AutoCloseable buildConsumers(int numberOfConsumers,
+                                            boolean syncCommit,
```

Comment:
In this case the consumers don't use consumer offsets, so `syncCommit` must be "false" to avoid error